### PR TITLE
chore: enable extensions for the project

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -75,4 +79,9 @@ pipeline {
         }
       }
     }
+    post {
+                always {
```
Comment (jglick) [inline Jenkinsfile]: Why the weird indentation?

Comment (pmuir) [reply Jenkinsfile]: I haven't worked out how to autoformat a Jenkinsfile in IntelliJ yet.

**Pull request merged**